src/Relation/Unary/Properties.agda

```diff
@@ -200,6 +201,10 @@ U-Universal = λ _ → _
 ------------------------------------------------------------------------
 -- Decidability properties
 
+≐? : {P : Pred A ℓ₁} {Q : Pred A ℓ₂} →
+     P ≐ Q → Decidable P → Decidable Q
+≐? (P⊆Q , Q⊆P) P? x = map′ P⊆Q Q⊆P (P? x)
```

Comment:
Two comments about this lemma:
* name: not quite sure about the name, but I can't think of the better at the moment
* definition: Can this be more 'simply' (compactly) expressed in terms of `Dec.map`? Delegation back to the conceptually prior, and differently 'simpler' `map′` might feel like a win, but as we are already in a (module-)scope in which `Dec.map` is defined and available, it seems 'better' to use it? (the heavy use of scare quotes is intentional in drawing attention to my ambivalence about the range of choices here)

The second point amounts really to saying: this is the 'obvious' lifting to `Unary` predicates of `Dec.map` on `Nullary` propositions... so "for the casual reader" it might be better to make that evident...?

Reply:
yeah... not sure what name I'd choose, but fortunately, blocking on the v2.1 release means that we collectively have plenty of time to think about this ;-)

Reply:
I get strong `map` vibes from this. The current names makes me think that I'll be deciding an equality between functions.

Reply:
let's go with `map` then